systemtests/src/test/java/io/enmasse/systemtest/marathon/MarathonTestBase.java

```diff
@@ -310,9 +310,16 @@ void doTestCreateDeleteAddressesViaAgentLong(AddressSpace addressSpace) throws E
         ArrayList<Destination> addresses = generateQueueTopicList("via-web", IntStream.range(0, addressCount));
 
         runTestInLoop(30, () -> {
-            consoleWebPage.createAddressesWebConsole(addresses.toArray(new Destination[0]));
-            consoleWebPage.deleteAddressesWebConsole(addresses.toArray(new Destination[0]));
-            Thread.sleep(5000);
+            try {
```

Comment:
I think we should add there this lines

```
selenium.saveScreenShots("testClass", "testName");
selenium.tearDownDrivers()
```

WDYT?

Reply:
Yes, youre right man, as 'onfailed' will not be called in this situation anymore, good call. 
I have added them